src/Stats.LogInterpretation/ClientNames.cs

```diff
@@ -0,0 +1,117 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Stats.LogInterpretation
+{
+    public static class ClientNames
+    {
+        public static IList<string> NuGet = new List<string>()
```

Comment:
Can these be readonly?

```suggestion
        public static readonly IReadOnlyList<string> NuGet = new List<string>
```